EventsExpress/ClientApp/src/components/event/edit-event-new.js

```diff
@@ -0,0 +1,80 @@
+﻿import React, { Component } from 'react';
+import EventForm from './event-form';
+import { connect } from 'react-redux';
+import { getFormValues, reset } from 'redux-form';
+import { setEventError, setEventPending, setEventSuccess, draft_event } from '../../actions/event-add-action';
+import { validateEventForm } from '../helpers/helpers'
+import { resetEvent } from '../../actions/event-item-view';
+import get_categories from '../../actions/category/category-list';
+import L from 'leaflet';
+
+class EditEventNew extends Component {
```

Comment:
Please, do not name components '-new'. Rename it to show functionality appropriately.